### PR TITLE
Fix deprecated Qt 5.15 usage

### PR DESCRIPTION
**Pull request by DonLakeFlyer**

### CODE REVIEW

File: src/Airmap/AirMapWeatherInfoManager.cc [hunk 1]
```diff
@@ -10,9 +10,8 @@
 #include "AirMapWeatherInfoManager.h"
 #include "AirMapManager.h"
 
-
-#define WEATHER_UPDATE_DISTANCE 50000                   //-- 50km threshold for weather updates
-#define WEATHER_UPDATE_TIME     30 * 60 * 60 * 1000     //-- 30 minutes threshold for weather updates
+#define WEATHER_UPDATE_DISTANCE     50000           //-- 50km threshold for weather updates
+#define WEATHER_UPDATE_TIME_MSECS   30 * 60 * 1000  //-- 30 minutes threshold for weather updates
```
Comment (DonLakeFlyer) [inline src/Airmap/AirMapWeatherInfoManager.cc:14]: As far as I can tell this value was incorrect. It is supposed to be milliseconds.

File: src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp [hunk 2]
```diff
@@ -186,9 +186,9 @@ bool QJsonWebToken::setAlgorithmStr(QString strAlgorithm)
 	// set algorithm
 	m_strAlgorithm = strAlgorithm;
 	// modify header
-	m_jdocHeader = QJsonDocument::fromJson(QObject::trUtf8("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
+    m_jdocHeader = QJsonDocument::fromJson(QStringLiteral("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
```
Comment (DonLakeFlyer) [inline src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp:189]: This doesn't seem to need to use `tr`? Correct?

Comment (jennerl) [reply src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp:189]: what's the motivation to change it?

Comment (jennerl) [reply src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp:189]: I've never touched this code...AirMap didn't author it

Comment (DonLakeFlyer) [reply src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp:189]: Oh, that's right. This is Gus' code. The calls in this pull are deprecated in Qt 5.15 and won't compile without warnings. These changes get rid of the warnings. This all should be fine then.

**Pull request merged**